App.js

```diff
@@ -56,8 +56,13 @@ class MonthView extends React.Component {
     constructor(props) {
         super(props);
 
+        let currentDate = new Date();
```

Comment:
Just a reminder that "new Date()" returns the current date in UTC, not in local time or a given venue's time zone.  I don't currently see any issues regarding that, but let's make sure to check the month / year boundaries when sorting and submitting documents by email.

Reply:
Is this correct? From a cursory console test in Chrome, it seems to return the local time. It's possible that this is Chrome-specific but Node returns in UTC, but from what I recall, that's not the case.

Reply:
My bad, you're right.  I guess I'm just being paranoid from when I had to convert the UTC dates in the database to local time to make the month strings, and from when I was originally using getTime() to determine the current date for pulling a day of events from the database.